core/src/main/java/feast/core/job/task/CreateJobTask.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.core.job.task;
+
+import feast.common.logging.AuditLogger;
+import feast.common.logging.entry.LogResource.ResourceType;
+import feast.core.job.JobManager;
+import feast.core.model.Job;
+import feast.core.model.JobStatus;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.event.Level;
+
+/** Task that starts recently created {@link Job} by using {@link feast.core.job.JobManager}. */
+@Slf4j
```

Comment:
Is this annotation used?

Reply:
Yes [here](https://github.com/mrzzy/feast/blob/9ff000e4391ffa6be6bb5496aa85719d492adae1/core/src/main/java/feast/core/job/task/CreateJobTask.java#L45)